templates/catalog_delete.html

```diff
@@ -0,0 +1,17 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <title>Delete catalogs</title>
+</head>
+<div>
+    <form method="post">{% csrf_token %}
+        <h4>Are you sure you want to delete the catalog: "{{ object.name }}"?</h4>
+        <button type="submit" class="btn btn-danger">Delete</button>
+        <a href="{{ view.get_success_url }}" class="btn btn-secondary">Cancel</a>
+    </form>
+</div>
+</html>
```

Comment:
This template should inherit the base template

Reply:
Done!